.github/workflows/deployment-test.yml

```diff
@@ -19,6 +19,7 @@ on:
       - ready_for_review
       - labeled
       - auto_merge_enabled
+  merge_group:
```

Comment:
if we plan on deployment test to be required for merge, I think the pre-checks should be updated as well

Reply:
If we remove the pre checks, we should add the following:

```yaml
concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}
  cancel-in-progress: true
```